internals/webpack/webpack.base.babel.js

```diff
@@ -88,4 +88,9 @@ module.exports = (options) => ({
   target: 'web', // Make web variables accessible to webpack, e.g. window
   stats: false, // Don't show stats in the console
   progress: true,
+
+  // Polling for vagrant based development
```

Comment:
Yeah I agree, was going to mention that but forgot. This should definitely not be included by default!